docs/ref/models/expressions.txt

```diff
@@ -252,6 +252,16 @@ The ``Func`` API is as follows:
         A class attribute that denotes the character used to join the list of
         ``expressions`` together. Defaults to ``', '``.
 
+    .. attribute:: arity
```

Comment:
I don't mind the term arity, but I've had experience with the term before. "num_expressions" is more accessible, but it's also less descriptive. Number of expressions for what? Arity has a very precise definition. "num_arguments" would be better than "num_expressions" I think, so I'd be open to that change.

Remember also that this attribute is for class designers and library authors to better validate their arguments. It's totally opt in. And if you're opting in you're probably reading the documentation that has the definition of the attribute anyway.

I'd also like a quick definition as a comment next to the base attribute, which I'll mark up separately.